### ISSUE TITLE
X doesn't start after a fresh install

### ISSUE THREAD

**Issue by johnodon** [MizterB/RetroPie-Setup-Ubuntu#9]
A few of us on the RP forum have experienced this.

Perform a vanilla install.  Run the script.  After first reboot, we see the Pacman plymouth theme, the screen goes black for about 20 seconds and we are dropped to a prompt.

running 'startx' from command prompt works as expected.

One of the fixes suggested was to update to a newer kernel but this has the possibility of introducing some other ill effects.

The fix that I have found is to preface the call to 'startx' with 'exec' in the .bash_profile file:

```
if [[ -z $DISPLAY ]] && [[ $(tty) = /dev/tty1 ]]; then
    exec startx -- >/dev/null 2>&1
fi
```

I'd like some others that have had this issue test this solution because I am still seeing the ~20 second delay before ES launches.

John

**Comment by johnodon** [comment]
PR submitted.  Once tested/merged, this can be closed.

**Comment by MizterB** [comment]
Personally, I don't have any load time issues in my setup with the current command.  However, I can confirm that this change does not negatively impact my startup times.  Seems identical.  Let's see what other might have to say.

**Comment by johnodon** [comment]
> Personally, I don't have any load time issues in my setup with the current command. However, I can confirm that this change does not negatively impact my startup times. Seems identical. Let's see what other might have to say.

Thanks.  I know that etheling and one other person on the forum saw the same issue I did.

**Comment by johnodon** [comment]
I've now tested this on a second machine (Lenovo T430).  Stock script doesn't start X and drops me at a prompt.  Adding the 'exec' corrected the issue same as my other system.

**Comment by MizterB** [comment]
Since this is affecting multiple people and doesn't cause any new issue as best I can tell, I've added the change to the mainline script.  Fixed in 61d9c8ed3cd35205e5bcdb6a7579a3b3ccf64eb3. Ignore the commit message :)
